sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -2817,6 +2831,15 @@ pi_result piContextRelease(pi_context Context) {
   return ContextReleaseHelper(Context);
 }
 
+// This function will initialize batch sizes for the compute anc copy command
+// lists
+static void initializeQueueBatchSizes(pi_queue Queue) {
```

Comment:
please move _pi_queue ctor implementation to pi_level_zero.cpp, and remove this function

Reply:
Done. Thanks.